src/components/PrimaryNav/PrimaryNavLink/PrimaryNavLink.tsx

```diff
@@ -0,0 +1,37 @@
+import { Icon } from "@canonical/react-components";
+import classNames from "classnames";
+import { NavLink } from "react-router-dom";
+
+export enum TestId {
+  PRIMARY_NAV_LINK = "primary-nav-link",
+}
+
+type PrimaryNavLinkProps = {
+  children?: React.ReactNode;
+  to: string;
+  iconName: string;
+  title: string;
+};
+
+const PrimaryNavLink = ({
```

Comment:
This component is identical to the one we used in Audit logs feature. I thought that doing it this way would make it easier for us to merge this branch and the `audit-logs` branch in the future. Let me know if it would be better to remove it.

Reply:
That makes sense. It would be great to make this change in main as well to make it easier to merge them both in the future.